data/data/bootstrap/files/usr/local/bin/approve-csr.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+KUBECONFIG="${1}"
+
+echo "Approving all CSR requests until bootstrapping is complete..."
+while [ ! -f /opt/openshift/bootkube.done ]
+    oc --config="$KUBECONFIG" get csr --no-headers | grep Pending | \
+        awk '{print $1}' | \
+        xargs --no-run-if-empty oc --config="$KUBECONFIG" adm certificate approve
```

Comment:
What do these CSRs look like?  Is there any simple way to be more strict on what is approved?